docs/source/architecture/contracts.md

```diff
@@ -21,14 +21,14 @@ For a guide of how to deploy these contracts automatically, see the [Deployment
 2. Deploy `StakingEscrow` with a dispatcher targeting it
 3. Deploy `PolicyManager` with its own dispatcher, also targeting it
 4. Deploy `Adjudicator` with a dispatcher
-5. Deploy `WorkLock` contract
-6. Transfer reward tokens to the `StakingEscrow` contract. These tokens are future mining rewards and initial allocations
-7. Run the `initialize()` method to initialize the `StakingEscrow` contract
-8. Set the address of the `PolicyManager` contract  in the `StakingEscrow` by using the `setPolicyManager(address)`
-9. Set the address of the `Adjudicator` contract  in the `StakingEscrow` by using the `setAdjudicator(address)`
-10. Set the address of the `WorkLock` contract  in the `StakingEscrow` by using the `setWorkLock(address)`
-11. Transfer tokens for distribution to the `WorkLock` contract
-12. Deploy `StakingInterface` with `StakingInterfaceRouter` targeting it
+5. Deploy `StakingInterface` with `StakingInterfaceRouter` targeting it
+6. Deploy `WorkLock` contract
+7. Set the address of the `PolicyManager` contract  in the `StakingEscrow` by using the `setPolicyManager(address)`
+8. Set the address of the `Adjudicator` contract  in the `StakingEscrow` by using the `setAdjudicator(address)`
```

Comment:
This two steps currently happen immediately after deploying the corresponding dispatcher. So it's:
3. Deploy PolicyManager and dispatcher
4. SetPolicyManager
5. Deploy Adjudicator and dispatcher
6. SetAdjuticator

In principle, it doesn't matter, but perhaps we should change it to reflect our current workflow.

Reply:
✔️